ds-collector-tests/cluster-dse-k8s.make

```diff
@@ -8,16 +8,21 @@ test:
 	echo "" >> /tmp/datastax/test-collector-k8s-dse.conf
 	echo "cqlshUsername=$$(kubectl -n cass-operator get secret cluster2-superuser -o yaml | grep " username" | awk -F" " '{print $$2}' | base64 -d && echo "")" >> /tmp/datastax/test-collector-k8s-dse.conf
 	echo "cqlshPassword=$$(kubectl -n cass-operator get secret cluster2-superuser -o yaml | grep " password" | awk -F" " '{print $$2}' | base64 -d && echo "")" >> /tmp/datastax/test-collector-k8s-dse.conf
-	../ds-collector/ds-collector -T -f /tmp/datastax/test-collector-k8s-dse.conf -n cluster2-dc1-default-sts-0
-	../ds-collector/ds-collector -X -f /tmp/datastax/test-collector-k8s-dse.conf -n cluster2-dc1-default-sts-0
+	./collector/ds-collector -T -f /tmp/datastax/test-collector-k8s-dse.conf -n cluster2-dc1-default-sts-0
+	./collector/ds-collector -T -p -f /tmp/datastax/test-collector-k8s-dse.conf -n cluster2-dc1-default-sts-0
+	./collector/ds-collector -X -f /tmp/datastax/test-collector-k8s-dse.conf -n cluster2-dc1-default-sts-0
 	if ! ls /tmp/datastax/ | grep -q ".tar.gz" ; then echo "Failed to generate artefacts in the K8s cluster "; ls -l /tmp/datastax/ ; exit 1 ; fi
 
 
 setup:
-	rm -f ../ds-collector/collect-info
-	cd ../ds-collector ; docker run --rm -v $$PWD:/volume -w /volume -t clux/muslrust rustc --target x86_64-unknown-linux-musl rust-commands/*.rs ; cd -
-	test -f ../ds-collector/collect-info
 	mkdir -p /tmp/datastax && rm -fr /tmp/datastax/*
+	# make diagnostics bundle
+	cd ../ ; ISSUE="TEST-$$(git rev-parse --abbrev-ref HEAD)--$$(git rev-parse --short HEAD)" make
```

Comment:
Not sure if its the branch name, but this causes a failure later in the sed : sed: 1: "s/\#issueId=.*/issueId= ...": bad flag in substitute command: 'h'

Reply:
odd. not in the github actions. i guess you are seeing this locally…? on macos? 
( i noticed in github actions that `$(git rev-parse --abbrev-ref HEAD)` is always `HEAD`. so maybe when the branch name actually comes in, the `/` slash crashes the sed…

Reply:
should be fixed with https://github.com/datastax/diagnostic-collection/pull/90/commits/7a32ad8e4d6a2bd67511ab3c83eae2f8c4312af6

(this is a good bug to catch)

Reply:
Yes, I checked out the branch and was running the tests locally with a make -f cluster-dse-k8s.make

Reply:
try now…

Reply:
that now works - not sure if there are other characters we need to be wary of on this, but more of an edge case at that point.